src/main/kotlin/com/openlattice/organizations/HazelcastOrganizationService.kt

```diff
@@ -460,78 +460,6 @@ class HazelcastOrganizationService(
 
     }
 
-    private fun fixOrganizations() {
```

Comment:
😍

Reply:
Wait until we have to fix organizations again ;)